### PR TITLE
Added Leveraged Tokens on Protocol Seeds

### PR DESCRIPTION
**Pull request by djphillyg**
Rewrote `ERC20PositionHelper` to support creation of leveraged/short tokens.

### CODE REVIEW

File: test/tests/js/TestSeeds.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ contract('Margin', () => {
   });
 });
 
+
```
Comment (BrendanChou) [inline test/tests/js/TestSeeds.js]: remove extra space

File: test/helpers/ERC20PositionHelper.js [hunk 2]
```diff
@@ -2,34 +2,38 @@ const { createOpenTx } = require('./MarginHelper');
 const { BIGNUMBERS } = require('./Constants');
 const BigNumber = require('bignumber.js');
 const { issueAndSetAllowance } = require('./TokenHelper');
-
```
Comment (BrendanChou) [inline test/helpers/ERC20PositionHelper.js]: Leave in space between functions/classes and contracts

File: test/helpers/ERC20PositionHelper.js [hunk 3]
```diff
@@ -2,34 +2,38 @@ const { createOpenTx } = require('./MarginHelper');
...
 const TokenProxy = artifacts.require('TokenProxy');
 const WETH9 = artifacts.require("WETH9");
 const EthWrapperForBucketLender = artifacts.require("EthWrapperForBucketLender");
 const { ZeroExProxyV1 } = require('../contracts/ZeroExV1');
 
-const { ADDRESSES } = require('./Constants');
+const { ADDRESSES, POSITION_TYPE } = require('./Constants');
 const { createSignedV1Order } = require('./ZeroExV1Helper');
-const HeldToken = artifacts.require("TokenA");
+// const HeldToken = artifacts.require("TokenA");
```
Comment (BrendanChou) [inline test/helpers/ERC20PositionHelper.js]: remove rather than comment out

File: test/helpers/ERC20PositionHelper.js [hunk 4]
```diff
@@ -38,122 +41,218 @@ async function createShortToken(
...
+        bucketLender.address,
+      ),
+    ]);
+    await bucketLender.deposit(
+      accounts[7],
+      new BigNumber('10').times(principal),
+      { from: accounts[7] },
+    );
+    return;
+  }
```
Comment (BrendanChou) [inline test/helpers/ERC20PositionHelper.js]: you might want to set this up as a switch statement or an if/else-if/else block where we throw some sort of error when type is not set. And anywhere else where we check type

File: test/helpers/ERC20PositionHelper.js [hunk 5]
```diff
@@ -38,122 +41,218 @@ async function createShortToken(
...
+      openTx.trader,
+      [DutchAuctionCloser.address],
+      [ERC20PositionWithdrawer.address],
+    );
+  }
+}
+
+function generateBuySellOrders(
+  accounts,
+  MakerToken,
```
Comment (BrendanChou) [inline test/helpers/ERC20PositionHelper.js]: Should these be in an object

**Comment by djphillyg** [comment]
i utilized the same test that was used to test these before.

**Pull request merged**